src/java_bytecode/java_object_factory.cpp

```diff
@@ -20,6 +20,11 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <linking/zero_initializer.h>
 
+#include <ansi-c/c_types.h>
+#include <ansi-c/string_constant.h>
+
+#include <goto-programs/goto_functions.h>
+
```

Comment:
For all of the above headers: why are they necessary, especially in the Java code?